lib/src/Base/Stat/SampleImplementation.cxx

```diff
@@ -2352,13 +2351,28 @@ void SampleImplementation::exportToCSVFile(const FileName & filename,
     }
     csvFile << "\n";
   }
+
+  const UnsignedInteger precision = ResourceMap::GetAsUnsignedInteger("Sample-CSVPrecision");
+  const String format = ResourceMap::Get("Sample-CSVFormat");
```

Comment:
It is a good idea to allow users to tweak the precision and format, but couldn't they be specified as arguments (with default values given by ResourceMap) of `Sample::exportToCSVFile`?

Reply:
that would be too much arguments to provide, these are for less frequentl use-cases (and we dont have keyword arguments)

Reply:
yes, we can have arguments